synapse/cores/common.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, link, store, **conf):
 
         logger.debug('Initializing Cortex')
 
+        self.on('node:add', self._onAddFifo, form='syn:fifo')
```

Comment:
We will also want to lift all the syn:fifo notes ons startup and call .gen() for them